crates/biome_service/src/workspace.rs

```diff
@@ -109,13 +109,20 @@ pub enum ServiceNotification {
     WatcherStopped,
 }
 
+fn is_false(value: &bool) -> bool {
+    !*value
+}
+
 #[derive(Debug, serde::Serialize, serde::Deserialize)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase")]
 pub struct SupportsFeatureParams {
     pub project_key: ProjectKey,
     pub path: BiomePath,
     pub features: FeatureName,
+
+    #[serde(default, skip_serializing_if = "is_false")]
+    pub ignore_includes: bool,
```

Comment:
Calling it `ignore_includes` is a bit reductive, because we also check VCS-ignored files. I suggest a more generic a more generic name e.g. `skip_ignore_check`

Reply:
What do you think of this suggestion @tt-a1i ?